tests/unit/AddCourseCard.spec.js

```diff
@@ -0,0 +1,21 @@
+import { mount } from "@vue/test-utils";
+import AddCourseCard from "../../src/resources/app/components/AddCourseCard";
+
+// every test file should include AT LEAST ONE "describe()", "it()", and "expect()"
```

Comment:
should be only appears on the example test, I don't think it is necessary to place this one here